dev-docs/source/PyCharm.rst

```diff
@@ -190,7 +190,7 @@ To use the remote debugger:
 Setting up PyCharm on Linux
 ###########################
 
-1. Use the native python interpreter (``/usr/bin/python2.7``) rather than from ``<Mantid Source Directory>/external/src/ThirdParty/lib/python2.7/python.exe``
+1. Use the native python interpreter (``/usr/bin/python3``) rather than from ``<Mantid Source Directory>/external/src/ThirdParty/lib/python3/python.exe``
```

Comment:
`/usr/bin/python3` is correct but the latter link needs correcting the same as above.